packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -293,6 +297,16 @@ https://github.com/shorebirdtech/shorebird/issues/472
         return ExitCode.software.code;
       }
     }
+
+    ReleaseArtifact? releaseAabArtifact;
+    try {
+      releaseAabArtifact =
+          await codePushClient.getAabArtifact(releaseId: release.id);
+    } catch (error) {
+      // Do nothing for now, not all releases will have an associated aab
```

Comment:
Maybe we should include a todo to introduce an error in the future once most of our releases have app bundles?

Reply:
Done.